website/views.py

```diff
@@ -102,13 +102,15 @@ def post(self, request):
 				last_name = userForm.cleaned_data['last_name']
 				photo = profileForm.cleaned_data['photo']
 				bio = profileForm.cleaned_data['bio']
+				token = profileForm.cleaned_data['token']
```

Comment:
In general - why are we storing the cleaned data into a local variable, and then storing the local into the 'real' object? Why not just assign the cleaned values directly to the object?